Config/database.php

```diff
@@ -0,0 +1,5 @@
+<?php
+$dbservername = 'localhost';
```

Comment:
These are my local settings. We can setup a way to do this without committing it to code.